source-code/ide-extension/tsup.config.ts

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from "tsup"
```

Comment:
I would use pluginBuildConfig from  @inlang/core/plugin to stay compliant.

Reply:
@samuelstroschein 😄 You missed to export the types correctly from package.json, therefore the import couldn't be found and we didn't use it "in that way already" in the codebase, you imported it from the file directly for the tests. I will fix that tomorrow. 🧼

Reply:
The types are exported correctly for node16 module resolution^^

I guess the ide extension (as well as sveltekit) don't use node16 module resolution. All of this module discrepancy is such a pain.

Reply:
I can take a look at this. I think I now have seen all edge cases when it comes to exporting types for all possible formats ^^